code/modules/antagonists/pirate/pirate_gangs.dm

```diff
@@ -205,3 +205,25 @@ GLOBAL_LIST_INIT(heavy_pirate_gangs, init_pirate_gangs(is_heavy = TRUE))
 	response_too_late = "You were not ready then, and now that time has passed. We can only go forward, never back."
 	response_not_enough = "You have insulted us, but there shall be no feud, only swift justice!"
 	announcement_color = "purple"
+
+//medieval militia, from OUTER SPACE!
+/datum/pirate_gang/medieval
+	name = "Medieval Warmongers"
+
+	is_heavy_threat = TRUE
+	ship_template_id = "medieval"
+	ship_name_pool = "medieval_names"
+
+	threat_title = "HOMAGE PAYMENT REQUEST"
+	threat_content = "SALUTATIONS, THIS IS %SHIPNAME AND WE ARE COLLECTING MONEY \
+		FROM THE VASSALS IN OUR TERRITORY, YOU JUST SO HAPPEN TO BE IN IT TOO!! NORMALLY \
+		WE WOULD SLAUGHTER ALL OF YOU AND PILLAGE YOUR BELONGINGS, BUT WE ARE WILLING \
```

Comment:
Why would they slaughter and pillage their own vassals? This is a Feudalism Fail

Reply:
i wanted to mean it as if you were "intruding in their territory" and thats why they had reasons to steal your shit